GPU/TPCFastTransformation/TPCFastTransform.h

```diff
@@ -253,6 +272,14 @@ class TPCFastTransform : public FlatObject
   float mTOFcorr;
 
   float mPrimVtxZ; ///< Z of the primary vertex, needed for the Time-Of-Flight correction
+
+#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE)
+  /// Note: if this is defined above, it will give errors during the GPU reconstruction?!
+  /// Correction of (x,u,v) with tricubic interpolator on a regular grid
+  std::unique_ptr<o2::tpc::SpaceCharge<float>> mCorrectionSlow; ///<! reference space charge corrections
+  bool mUseSlowCorrection{false};                               ///<! flag whether to use the slow correction
+#endif
```

Comment:
This breaks GPU processing, since it relies on the fact that memory layouts on host and GPU are identical.
You must not have data members that exist only on the GPU.
The GPU should not use bool in shared data members, you can use char instead.
I also don't like that the flat object has a unique ptr.